tonic/models/flo2d/utils.py

```diff
@@ -0,0 +1,140 @@
+from osgeo import osr
+import mpl_toolkits.basemap.pyproj as pyproj
+
+import numpy as np
+import pandas as pd
+
+from scipy import stats
+import quantities as pq
+from scipy.spatial import cKDTree
+
+# Default projection from Alan Hamlet 2015-06-29
+default_proj = pyproj.Proj('+proj=lcc +lat_1=47.5 +lat_2=48.73333333333333 '
+                           '+lat_0=47'' +lon_0=-120.8333333333333 '
+                           '+x_0=609601.2192024385 +y_0=0 +datum=NAD83 '
+                           '+units=us-ft')
+
+
+def esriprj2standards(shapeprj_path, kind='proj4'):
+    '''Return string representation of projection in shapeprj_path.
+
+    Parameters
+    ----------
+    shapeprj_path : str
+        Filepath to ESRI style projection file.
+    kind : str {'prj', 'wkt', 'proj4', 'epsg'}
+        Projection standard.
+
+    Returns
+    ----------
+    projection_params : str
+        Projection parameters.
+    '''
+    prj_file = open(shapeprj_path, 'r')
+    prj_txt = prj_file.read()
+    srs = osr.SpatialReference()
+    srs.ImportFromESRI([prj_txt])
+    if kind.lower() == 'prj':
+        return prj_txt
+    if kind.lower() == 'wkt':
+        return srs.ExportToWkt()
+    elif kind.lower() == 'proj4':
+        return srs.ExportToProj4()
+    elif kind.lower() == 'epsg':
+        srs.AutoIdentifyEPSG()
+        return srs.GetAuthorityCode(None)
+    else:
+        raise ValueError('Kind is unknown')
```

Comment:
`basemap` can plot shapefiles but it's support is pretty limited.

This function just returns a string describing a shapefiles projection.  This is useful if you want to know how to setup a `basemap` plot or it can be fed into a `proj4` object for reprojecting data (this is my use case).